packages/flutter/lib/src/material/app.dart

```diff
@@ -379,11 +428,14 @@ class _MaterialAppState extends State<MaterialApp> {
   }
 
   // Combine the Localizations for Material with the ones contributed
-  // by the localizationsDelegates parameter, if any.
+  // by the localizationsDelegates parameter, if any. Only the first delegate
```

Comment:
put a blank line after the first sentence. We use the first paragraph for short docs in various places.

Reply:
never mind, just realised this is private